homeassistant/components/zwave.py

```diff
@@ -88,6 +95,32 @@ def _obj_to_dict(obj):
             if key[0] != '_' and not hasattr(getattr(obj, key), '__call__')}
 
 
+def _node_name(node):
+    """ Returns the name of the node. """
+    return node.name or "{} {}".format(
+        node.manufacturer_name, node.product_name)
+
+
+def _value_name(value):
+    """ Returns the name of the value. """
+    return "{} {}".format(_node_name(value.node), value.label)
+
+
+def _object_id(value):
```

Comment:
I am fine with this being extracted. As far as I can see, `_object_id` is needed to determine the lookup key for specific node config.